billcom_integration/models/account_move.py

```diff
@@ -0,0 +1,1084 @@
+# Copyright 2025 Binhex - Simple Solutions
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMove(models.Model):
+    _name = "account.move"
+    _inherit = ["account.move", "billcom.abstract.model"]
+
+    # Bill.com specific fields for bills/invoices
+    billcom_status = fields.Char(
+        string="Bill.com Status",
+        help="Payment status from Bill.com (PAID, UNPAID, PARTIALLY_PAID, etc.)",
+        readonly=True,
+    )
+    billcom_document_ids = fields.One2many(
+        "billcom.document",
+        "bill_id",
+        string="Bill.com Documents",
+        help="Documents attached to this bill in Bill.com",
+    )
+    billcom_invoice_number = fields.Char(
+        string="Bill.com Invoice Number",
+    )
+    billcom_payment_link = fields.Char(
+        string="Bill.com Payment Link",
+        help="Payment link for customer to pay invoice online",
+        readonly=True,
+        copy=False,
+    )
+
+    active = fields.Boolean()
+
+    def _prepare_bill_data(self, for_bulk=False):
+        """Prepare bill data for Bill.com API
+
+        Args:
+            for_bulk: If True, includes additional fields required for bulk endpoint
+        """
+        self.ensure_one()
+        if not self.is_sync_to_billcom or not self.partner_id.is_sync_to_billcom:
+            return False
+
+        if self.move_type != "in_invoice":
+            return False
+
+        # Get billcom.item model for tax mapping
+        item_model = self.env["billcom.item"]
+
+        # Prepare line items
+        lines = []
+        for line in self.invoice_line_ids:
```

Comment:
```suggestion
        for line in self.invoice_line_ids.filtered(lambda it: it.display_type == "product"):
```
Don't you need to exclude line notes or sections?